api/app/signals/apps/signals/management/commands/load_categories.py

```diff
@@ -0,0 +1,62 @@
+import json
+
+from django.core.management import BaseCommand
+
+from signals.apps.signals.models import Category
+
+
+class Command(BaseCommand):
+    data = None
+    slugsdict = {}
+
+    def _init_slugs_dict(self) -> None:
+        if self.data is not None:
+            for cat in self.data:
+                self.slugsdict[cat['pk']] = cat['fields']['slug']
+
+    def _get_parent(self, id) -> Category:
+        if id in self.slugsdict:
+            return self._get_cat(slug=self.slugsdict[id])
+        else:
+            return None
+
+    def _get_cat(self, slug) -> Category:
+        try:
+            return Category.objects.get(slug=slug)
+        except Exception:
+            return None
+
+    def add_arguments(self, parser) -> None:
+        parser.add_argument('data_file', type=str)
+
+    def handle(self, *args, **options) -> None:
+        self.processed_cats = set()
+        with open(options['data_file']) as f:
+            self.data = json.load(f)
+        if self.data is not None:
+            self._init_slugs_dict()
+            self.stdout.write(f'Total Categories: {len(self.data)}')
+            for c in self.data:
+                fields = c['fields']
+                parent_id = fields.pop('parent')
+                if parent_id:
+                    fields['parent'] = self._get_parent(parent_id)
```

Comment:
This is dependent on the order in the datafile, if the parent is processed after the child there will be no relation between those two categories.